sdks/python/tests/library_integration/openai/test_openai.py

```diff
@@ -492,3 +497,27 @@ async def async_f():
         assert len(fake_message_processor_.trace_trees) == 1
 
         assert_equal(EXPECTED_TRACE_TREE, fake_message_processor_.trace_trees[0])
+
+
+def test_openai_usage_keys_are_as_expected():
```

Comment:
@alexkuzmik this will break our build every time a vendor (open api) adds a new key to the response. I would not put this as part of tests and fail CI since its not actually a problem. we can get notified but not fail

Reply:
@Nimrod007 , got it, ok, makes sense. We'll create another workflow later for such notifications.